completions/fastfetch.fish

```diff
@@ -60,13 +60,21 @@ function __fastfetch_complete_structure
 end
 
 echo '
-import json, subprocess, sys
+import json
+import subprocess
+import sys
+
```

Comment:
No benefits but makes the script larger. Please revert this change and new empty lines below.